lib/chef/chef_fs/file_system/repository/chef_repository_file_system_root_dir.rb

```diff
@@ -110,6 +110,7 @@ def create_child(name, file_contents = nil)
               child_paths[name].each do |path|
                 begin
                   Dir.mkdir(path)
+                  File.chmod(0700, path) if name == "nodes"
```

Comment:
It also just hit me that these permission changes only work in unix land so I gotta go talk to some Windows folks.